app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/mnemonics/RestoreMnemonicsScreen.kt

```diff
@@ -74,15 +74,26 @@ fun RestoreMnemonicsScreen(
         state = state,
         onBackClick = viewModel::onBackClick,
         onSkipClicked = viewModel::onSkipClick,
```

Comment:
shall we better rename this to "onSkipSeedPhraseClick" and the viewmodel method so we can distinguish between main seed phrase and the rest of seed phrases?